### PR TITLE
Updating Readme

### PR DESCRIPTION
**Pull request by abbytiz**
@cftorres @aaron-prindle @nkubala

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,42 +154,38 @@ type MultiVersionInfo struct {
 
 ## Known issues
 
-To run container-diff on image IDs or URLs, docker must be installed.
+To run container-diff on image IDs, docker must be installed.
 
 ## Example Run
 
+// TODO FIX EXAMPLE RUN
```
Comment (aaron-prindle) [inline README.md]: What is this referring to?

**Comment by aaron-prindle** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (abbytiz) [inline README.md]: Oops, that was a note to self that I addressed but forgot to take out.  I did verify by rerunning the example and the output currently there is the most recent.

**Comment by aaron-prindle** [comment]
LGTM, merging

**Pull request merged**